src/contributor-guide/dod/automated-tests.md

```diff
@@ -0,0 +1,161 @@
+---
+group: dod
+title: DoD - Automated Tests
+---
+
```

Comment:
Does this doesn't require to mention about performance tests too?

Reply:
This PR is introducing TCO analysis and static tests coverage for BIC sections.
If you think performance test requirements should be added to DoD - please make your proposition in a separate PR.